edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/AgentEventIds.cs

```diff
@@ -28,9 +28,10 @@ public struct AgentEventIds
         public const int KubernetesPlanner = EventIdStart + 2200;
         public const int KubernetesCommand = EventIdStart + 2300;
         public const int KubernetesOperator = EventIdStart + 2400;
-        public const int LogsUploadRequestHandler = EventIdStart + 2500;
-        public const int ModuleClientProvider = EventIdStart + 2600;
-        public const int LogsRequestHandler = EventIdStart + 2700;
+        public const int KubernetesReporter = EventIdStart + 2500;
+        public const int LogsUploadRequestHandler = EventIdStart + 2600;
+        public const int ModuleClientProvider = EventIdStart + 2700;
+        public const int LogsRequestHandler = EventIdStart + 2800;
```

Comment:
Should we move the kubernetes constants into the kubernetes assembly instead of core?

Reply:
Yes we can.  I just need to make sure the EventIds are numerically unique.